llama-index-core/llama_index/core/workflow/checkpointer.py

```diff
@@ -0,0 +1,180 @@
+import asyncio
+import uuid
+import functools
+from llama_index.core.bridge.pydantic import (
+    BaseModel,
+    Field,
+    ConfigDict,
+)
+from typing import (
+    Optional,
+    Dict,
+    Any,
+    List,
+    Protocol,
+    TYPE_CHECKING,
+    Type,
+    Awaitable,
+)
+from llama_index.core.workflow.context import Context
+from llama_index.core.workflow.context_serializers import BaseSerializer, JsonSerializer
+from llama_index.core.workflow.handler import WorkflowHandler
+from .events import Event
+from .errors import *
```

Comment:
nit: I really dislike glob imports 😅

Reply:
i'll import the required errors more explicitly!